stdlib/test/builtin/test_simd.mojo

```diff
@@ -1546,6 +1546,41 @@ def test_contains():
     assert_false(0 in y or 5 in y)
 
 
+def test_comparison():
+    alias I = SIMD[DType.int32, 4]
+    var i = I(-2, 0, 2, 10)
+
+    assert_true(i.__lt__(I(-1, 1, 3, 11)).reduce_and())
+    assert_false(i.__lt__(I(-2, -1, 1, 9)).reduce_or())
+    var mixed_lt = i.__lt__((I(1, 1, 2, 9)))
+    assert_true(mixed_lt[0] and mixed_lt[1])
+    assert_false(mixed_lt[2] or mixed_lt[3])
+
+    assert_true(i.__le__(I(-2, 1, 2, 11)).reduce_and())
+    assert_false(i.__le__(I(-3, -1, 1, 9)).reduce_or())
+    var mixed_le = i.__le__((I(-2, 1, 1, 9)))
+    assert_true(mixed_le[0] and mixed_le[1])
+    assert_false(mixed_le[2] or mixed_le[3])
+
+    assert_true(i.__eq__(I(-2, 0, 2, 10)).reduce_and())
+    assert_false(i.__eq__(I(-3, 1, -2, 11)).reduce_or())
+    var mixed_eq = i.__eq__((I(-2, 0, 1, 9)))
```

Comment:
Could you please also add tests for `__ne__` here?

Reply:
yes, will add tests for `__ne__` as well.

Reply:
addressed in [60afbc7](https://github.com/modularml/mojo/pull/3221/commits/60afbc7ac33a406de178002660a561dcad6f462a)